### PR TITLE
Update: Add date filter value for selecting single date

### PR DESCRIPTION
**Pull request by mattgraham1995**
Splitting up PR #253 

image

Single date selection filter value

### CODE REVIEW

File: packages/reports/addon/components/filter-values/date.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import layout from '../../templates/components/filter-values/date';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['date'],
```
Comment (jkusa) [inline packages/reports/addon/components/filter-values/date.js]: `filter-values--date`

File: packages/reports/addon/components/filter-values/date.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  /**
+   * @property {Moment} selectedDate - local moment set by date picker
+   */
+  selectedDate: null,
```
Comment (jkusa) [inline packages/reports/addon/components/filter-values/date.js]: `_selectedDate`

File: packages/reports/addon/components/filter-values/date.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  /**
+   * @property {String} savedDate - the date that's saved in the filter
+   */
+  savedDate: computed.oneWay('filter.values.firstObject'),
```
Comment (jkusa) [inline packages/reports/addon/components/filter-values/date.js]: `_`

File: packages/reports/addon/components/filter-values/date.js (hunk 2 above)
Comment (jkusa) [reply packages/reports/addon/components/filter-values/date.js]: Do we need to intialize this to `filter.values.firstObject`?

File: packages/reports/app/styles/navi-reports/components/filter-values.less [hunk 4]
```diff
@@ -20,6 +20,49 @@
...
+        padding: 5px 10px;
+        width: 100%;
+      }
+
+      &__reset {
+        margin-right: 10px;
+      }
+    }
+
+    .ember-power-calendar-nav {
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/filter-values.less]: Is this used?

**Comment by jkusa** [comment]
Can we add a little padding to the top?

File: packages/reports/addon/components/filter-values/date.js (hunk 2 above)
Comment (mattgraham1995) [reply packages/reports/addon/components/filter-values/date.js]: Should it be considered private if its passed down to a child component?

File: packages/reports/addon/components/filter-values/date.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattgraham1995) [reply packages/reports/addon/components/filter-values/date.js]: Were you looking at an old commit? This classname is changed in the most recent commit

[the remark this answers is not in the document]
Comment (cythrawll) [reply packages/reports/addon/components/filter-values/date.js]: nah it used a comment that didn't submit like a week ago. Didn't see it when I hit approve.

**Pull request merged**